source/our_tools/tractoflow.rst

```diff
@@ -3,4 +3,220 @@
+            nextflow -c ../beluga.conf run ../tractoflow-*/main.nf --root ../input_data_tractoflow \
+                -with-singularity ../tractoflow_*.img -resume -with-report report.html \
+                --dti_shells "0 1500" --fodf_shells "0 1500" --step 0.5 --nbr_seeds 15 \
+                --wm_seeding false --mean_frf false --set_frf true
+
+    #. Before launching your command for real, let's test it quickly using an interactive node. You ask access to an interactive node doing this:
+
+        .. code-block:: bash
+
+            cd ~/scratch/Project_X
+            mkdir output_tractoflow
+            cd output_tractoflow
+
+            salloc -c 32 --mem 32G --time 00:10:00 -A def-descotea
+
+            # Wait for the node to be allocated to you.
+            # If our lab lacks priority and it is too long, you can try with -c 16 --mem 16Gb.
+            # When allocation is granted, you should see:
+            #   salloc: Pending job allocation xxx
+            #   salloc: job xxx queued and waiting for resources
+            #   salloc: job xxx has been allocated resources
+            #   salloc: Granted job allocation xxx
+            #   salloc: Waiting for resource configuration
+            #   salloc: Nodes yyym are ready for job
+
+            # Then type the tractoflow command with --help at the end. Tractoflow's help should be printed.
+
+            # Then, you may type the real command. Make sure it starts running. Once sure that it found the data, the img, the code, you can kill it by pressing ctrl-c.
+
+       If it fails:
+
+         - Is the data binding correctly (see point 5)?
+         - If one process fails, you should see a warning such as ``[11/53e26e] NOTE: Process `Bet_Prelim_DWI (101309)` terminated with an error exit status (127) -- Execution is retried (3)``. You can then check in the associated folder's log to see the error. For example:
+
+         .. code-block:: bash
+
+            ls -a work/11/53e26e*/  # Check that files are there
+            cat work/11/53e26e*/.command.log  # Check the error
+
+    #. Last decision to take on the cluster: Do you need 1 node or multiple nodes? Beyond 20 subjects or so, we recommend using multiple nodes. See the HPC part of tractoflow's `launch page <https://tractoflow-documentation.readthedocs.io/en/latest/pipeline/launch.html>`_. Depending on the cluster you are using (beluga, cedar, etc.), you have access to different types of nodes. Keeping the option "mem=0" in your sbatch (see next step) means you accept any node. If your data is very heavy (ex, HCP data), you might need to restric to the nodes with more RAM. See the `memory section here <https://docs.computecanada.ca/wiki/Running_jobs#Memory>`_ for more information.
+
+    #. We have all the ingredients to prepare the final sbatch. Here, we ask for 4 nodes, with 32 threads each and 116Gb of RAM each (see the -with-mpi option). Create a file cmd_my_tractoflow.sh with the following.
```

Comment:
Here it is 2 nodes and not 4. Please remove 116G of RAM and say instead all the RAM available on the node.